.github/workflows/build-images.yml

```diff
@@ -159,7 +159,7 @@ jobs:
       - name: "Setup python"
         uses: actions/setup-python@v5
         with:
-          python-version: 3.8
+          python-version: 3.9
```

Comment:
Nit: we should probably start quoting these no, so that `3.10` doesn't become `3.1` (`x: 3.10` vs `x: "3.10"`) :grin:

Reply:
Good point. Typical pitfall. On the other hand the `- name: "Setup python"` is not needing any quotes, this is a safe string :-D

Will wait for a full review cycle and then adjust as a bulk.

Reply:
Indeed. Great catch @ashb !

Reply:
Fixed